src/main/java/it/gov/pagopa/reward/service/reward/OnboardedInitiativesServiceImpl.java

```diff
@@ -33,7 +35,8 @@ public OnboardedInitiativesServiceImpl(HpanInitiativesRepository hpanInitiatives
     @Override
     public Flux<InitiativeConfig> getInitiatives(TransactionDTO trx) {
         if(OperationType.CHARGE.equals(trx.getOperationTypeTranscoded()) || isPositive(trx.getEffectiveAmount())){
-            return getInitiatives(trx.getHpan(), trx.getTrxChargeDate(), null);
+            return getInitiatives(trx.getHpan(), trx.getTrxChargeDate(), null)
+                    .map(Pair::getFirst); // TODO The refund's initiatives does not support family unit
```

Comment:
remove the "TODO" word, this is not a TODO, this is just a comment/remainder

Reply:
let's use the same comment used in other piece of code "Async trx support just physical person initiatives (not families)"